.tekton/gatekeeper-operator-3-17-push.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     build.appstudio.redhat.com/commit_sha: '{{revision}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
+    build.appstudio.openshift.io/build-nudge-files: "build/konflux-patch.sh"
```

Comment:
Neat! https://konflux-ci.dev/docs/how-tos/configuring/component-nudges/#customizing-nudging-prs

Do you know if this *adds* this file to the default patterns? It seems like it should, it would be a hassle to want to copy that documented default and then add our customization, but I want to check if you've confirmed the behavior.